src/main/java/roomescape/controller/UserRoomEscapeController.java

```diff
@@ -0,0 +1,59 @@
+package roomescape.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+import roomescape.Reservation;
+import roomescape.dto.request.ReservationCreateRequest;
+import roomescape.dto.response.ReservationResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+
+@Controller
+public class UserRoomEscapeController {
+
+    private static final AtomicLong autoIncrement = new AtomicLong(0);
+    private static final List<Reservation> reservations = new ArrayList<>();
+
+    @GetMapping("/reservations")
+    public ResponseEntity<List<ReservationResponse>> reservations() {
+        List<ReservationResponse> response = reservations.stream()
+                .map(ReservationResponse::from)
+                .toList();
+
+        return ResponseEntity.ok(response);
+    }
+
+    @PostMapping("/reservations")
+    public ResponseEntity<ReservationResponse> addReservations(@RequestBody ReservationCreateRequest request) {
+        Reservation newReservation = request.toDomain(autoIncrement.incrementAndGet());
+
+        reservations.add(newReservation);
+
+        return ResponseEntity.ok(ReservationResponse.from(newReservation));
```

Comment:
```suggestion
        ReservationResponse response = ReservationResponse.from(newReservation);
        return ResponseEntity.ok(response);
```
이러한 코드를 선호하는 편인데 어떠신가요?

Reply:
좋네요. 너무 생략되어 있었던 것 같아요.

[refactor: 너무 생략되어 읽기 어려운 코드를 리팩터링](https://github.com/woowacourse/spring-roomescape-admin/pull/246/commits/75497f7eb4159a3e59ca5bdaf043765dc14d95c2)